test/merchant_test.rb

```diff
@@ -6,8 +6,8 @@
 class MerchantTest < MiniTest::Test
   attr_reader :parent, :merch
   def setup
-    sales_engine = SalesEngine.new
-    @parent = MerchantRepository.new('test/support/merchants.csv', sales_engine)
+    sales_engine = SalesEngine.new('test/support')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.